cdap-metadata-ext-spanner/src/main/java/io/cdap/cdap/metadata/spanner/SpannerChangeRequest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2025 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.metadata.spanner;
+
+import com.google.cloud.spanner.Mutation;
+import io.cdap.cdap.spi.metadata.MetadataChange;
+import java.util.List;
+
+/**
+ * A simple class to pass around a list of Spanner mutations, along with the
+ * metadata change that it effects.
+ */
+public class SpannerChangeRequest {
```

Comment:
nit: this can be renamed to `ChangeRequest`. Since this is in `io.cdap.cdap.metadata.spanner` package, `Spanner` is redundant in the naming.